install_config/configuring_pipeline_execution.adoc

```diff
@@ -67,10 +67,17 @@ jenkinsPipelineConfig:
 
 [IMPORTANT]
 ====
-When a Pipeline build configuration is created,
-{product-title} instantiates the `*jenkinsPipelineConfig*` template *only if*
-no existing service name in the project matches the `*serviceName*` field.
-This means `*serviceName*` must be chosen such that it is unique in the project.
+When a Pipeline build configuration is created, {product-title} instantiates
+the `*jenkinsPipelineConfig*` template *only if* no existing service name in
+the project matches the `*serviceName*` field.  This means `*serviceName*` must
+be chosen such that it is unique in the project.
+====
+
+[IMPORTANT]
+====
+If you're using an external Jenkins server to manage your pipeline, you'll need
+to create a service for {product-title} to use it.
+xref:../dev_guide/integrating_external_services.adoc#dev-guide-integrating-external-services[Learn more about integrating external services.]
```

Comment:
I think we want to say more on this topic.  We are trying to communicate two things:

1) if you want to prevent autoprovisioning from happening, you can do so by creating a properly named service(and make it clear how to determine that name), which will block the autoprovisioning (because the service already exists).

2) one reason you might want to do (1) is because you already have a jenkins server that you want to use.  that server could be running in another project, or external to openshift entirely.  either way, you should point the service you create in fulfillment of (1), to that jenkins, using the external service mechanism.